lib/cn/node_split.c

```diff
@@ -24,11 +25,24 @@ struct reverse_kblk_iterator {
     uint32_t oi_offset;
 };
 
+struct forward_leaf_iterator {
```

Comment:
Add comment that this is a WBTree leaf iterator.  Or rename struct to indicate that.  As is, in a file called "node_split", on might think this was a cn tree leaf iterator.